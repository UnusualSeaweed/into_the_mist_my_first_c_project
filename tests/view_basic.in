view
view
place A1
view
view
term
