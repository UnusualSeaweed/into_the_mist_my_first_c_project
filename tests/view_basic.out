J10,.................................................
J10,.................................................
M3,..........................................xxxxxxx
M3,..........................................xxxxxxx
